docs/managed-plugin-upgrade-process.md

```diff
@@ -0,0 +1,27 @@
+## Steps to upgrade to the latest HPE 3PAR Managed Volume plugin
+
+1.	The current installation of the volume plugin with old version (e.g store/hpestorage/hpedockervolumeplugin:1.1) is assumed to be from docker store (https://store.docker.com/plugins/hpe-3par-docker-volume-plugin).
+2.	It is assumed that all volumes are mounted and I/O operations are in-progress.
+3.	Stop I/O operations and unmount all volumes.
+4.	Disable the plugin forcefully using below command:
+
+>*$ docker plugin disable --force PLUGIN*
+
+5.	Upgrade HPE 3PAR managed volume plugin which is available on docker store (https://store.docker.com/plugins/hpe-3par-docker-volume-plugin). 
+6.	Execute the below command to upgrade the plugin to the latest version.
+
+>*$ docker plugin upgrade PLUGIN [REMOTE]*
+
+>*For example, $ docker plugin upgrade store/hpestorage/hpedockervolumeplugin:1.1 store/hpestorage/hpedockervolumeplugin:2.0*
+
```

Comment:
1. you can mention about driver change from iscsi to FC 
2. docker plugin set <alias> / <tag> ..  (either for certs.source, glibc_libs.source=)